src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java

```diff
@@ -378,6 +356,15 @@ private void importFromGcs(String projectId, TableReference tableRef, @Nullable
             JobInfo.SchemaUpdateOption.ALLOW_FIELD_ADDITION.name(),
             JobInfo.SchemaUpdateOption.ALLOW_FIELD_RELAXATION.name()));
         }
+      } else {
+        if (!tableExists && Operation.UPSERT.equals(operation)) {
```

Comment:
nit: move else if in same line similar to:
https://github.com/data-integrations/google-cloud/blob/f054621188c1301d7d47b204b2d4888f6a7a08d3/src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSArgumentSetter.java#L157

Reply:
We dont need this check for insert?

Reply:
For INSERT , since it's a Big Query Job directly into destination table, it Automatically creates the Table.